torch/fx/symbolic_trace.py

```diff
@@ -173,6 +173,10 @@ def trace(self, root: Union[torch.nn.Module, Callable]) -> Graph:
         fn, args = self.create_args_for_root(fn, isinstance(root, torch.nn.Module))
 
         orig_call = torch.nn.Module.__call__
+        orig_getattr = torch.nn.Module.__getattr__
+
+        def module_getattr_wrapper(mod, attr):
+            return self.create_proxy('get_attr', attr, (), {})
```

Comment:
Two things I can think of:

1. I think you need to do this only for attributes that aren't submodules
2. The `target` you pass to `create_proxy` should be a fully-qualified name. The way it's written now will only work for attributes at the base of the module hierarchy